# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: security/sandbox/chromium/sandbox/win/src/eat_resolver.cc
# Commit: 370c40e49230
# Full Hash: 370c40e4923073a87845b57c1f4851bacd9663da
# Author: Bob Owen <bobowencode@gmail.com>
# Date: 2019-06-12 21:42:35
# Regressor Bug: 1552160
# File Overlap Count: 2
# Description:
#   Bug 1552160 Part 1: Roll-up of chromium sandbox update and mozilla patches to get a running browser. r=jld,aklotz,tjr,bobowen
#   
#   This updates security/sandbox/chromium/ files to chromium commit 84108231f6e6e0772fb9a4643679ce76aa771e67.
#   
#   Existing and new patches applied from security/sandbox/chromium-shim/patches/with_update/ to give a compiling and mostly working browser.
# ==============================================================================

diff -r bd5ecd72f76c -r 370c40e49230 security/sandbox/chromium/sandbox/win/src/eat_resolver.cc
--- a/security/sandbox/chromium/sandbox/win/src/eat_resolver.cc	Wed Jun 12 21:55:59 2019 +1200
+++ b/security/sandbox/chromium/sandbox/win/src/eat_resolver.cc	Wed Jun 12 11:10:48 2019 +0100
@@ -19,9 +19,9 @@
                                  void* thunk_storage,
                                  size_t storage_bytes,
                                  size_t* storage_used) {
-  NTSTATUS ret = Init(target_module, interceptor_module, target_name,
-                      interceptor_name, interceptor_entry_point,
-                      thunk_storage, storage_bytes);
+  NTSTATUS ret =
+      Init(target_module, interceptor_module, target_name, interceptor_name,
+           interceptor_entry_point, thunk_storage, storage_bytes);
   if (!NT_SUCCESS(ret))
     return ret;
 
@@ -30,7 +30,7 @@
 
 #if defined(_WIN64)
   // We have two thunks, in order: the return path and the forward path.
-  if (!SetInternalThunk(thunk_storage, storage_bytes, NULL, target_))
+  if (!SetInternalThunk(thunk_storage, storage_bytes, nullptr, target_))
     return STATUS_BUFFER_TOO_SMALL;
 
   size_t thunk_bytes = GetInternalThunkSize();
@@ -50,7 +50,7 @@
   *eat_entry_ = static_cast<DWORD>(reinterpret_cast<uintptr_t>(thunk_storage)) -
                 static_cast<DWORD>(reinterpret_cast<uintptr_t>(target_module));
 
-  if (NULL != storage_used)
+  if (storage_used)
     *storage_used = GetThunkSize();
 
   return ret;